src/components/footer/AboutModal.tsx

```diff
@@ -32,11 +32,11 @@ const AboutModal = (props: ModalProps): React.ReactElement => {
               <a href="https://developers.google.com/open-source/soc/" rel="noreferrer" target="_blank">
                 <img
                   alt="Google Summer of Code"
-                  src="https://summerofcode.withgoogle.com/static/img/summer-of-code-logo.svg"
+                  src="https://summerofcode.withgoogle.com/assets/media/logo.svg"
                   style={{ height: '2.5em' }}
                 />
               </a>
-              <a href="https://developers.google.com/open-source/gci/" rel="noreferrer" target="_blank">
+              <a href="https://opensource.google/static/images/documentation/reference/growing/GCI-logo.png" rel="noreferrer" target="_blank">
```

Comment:
Why did this link change?

Reply:
there is two logos  not appear well 
![image](https://user-images.githubusercontent.com/63047860/224389699-e4872609-be48-48c2-a6bb-1b2507ba6c7e.png)

Reply:
But you changed the link's href, not the image. That's a rather fundamental HTML thing to get wrong.

Also, you should put assets into the repo so we don't depend on 3rd parties.

Reply:
I am sorry it was a typo, i will update it